app/assets/javascripts/modules/calendar.es6

```diff
@@ -164,13 +164,18 @@ class Calendar extends TapBase {
   }
 
   eventRender(event, element) {
-    const now = moment();
+    const now = moment(),
+      elementId = `event_${event._id}`;
     let end = event.end.clone();
 
     // account for timezone weirdness
     if (end.subtract(1, 'h') < now) {
       element.addClass('fc--past');
     }
+
+    $(element).attr('id', elementId);
+
+    event.elementId = elementId;
```

Comment:
Storing an ID reference to the DOM element in the event object.

This is instead of storing the jQuery collection object (see appointment-availability.es6 below) which was causing issues with circular references in the tests.